drbg/src/hashdrbg.rs

```diff
@@ -0,0 +1,220 @@
+use crate::{
+    helpers::{slice_to_u896, truncate_from_start},
+    props::{
+        MAX_BYTES_ADDITIONAL_INPUT, MAX_BYTES_PERSONALIZATION_STRING, MAX_BYTES_PER_REQUEST,
+        MAX_REQUESTS_BETWEEN_RESEEDS,
+    },
+    HashDrbgProps,
+};
+use core::iter::once;
+use crypto_bigint::{prelude::ArrayEncoding, U896};
+pub use digest;
```

Comment:
`pub` here basically allows this to be a part of our public API, which I'm not sure is what you want. Are you intending to re-export this crate?

Reply:
I guess since this is not a public module, it doesn't end up re-exporting. It is still a tad weird I think, I guess I don't see why this is `pub`, and would prefer leaving it non-`pub` and only occasionally use `pub` if the intent for the statement is to export from the create in the end.

So just a nit.